build-tools/packages/build-tools/src/type-test-generator/main.ts

```diff
@@ -113,7 +113,7 @@ const testString: string[] = [
  * Generated by fluid-type-test-generator in @fluidframework/build-tools.
  */
 import type * as old from "${previousPackageName}";
-import type * as current from "../../index";
+import type * as current from "../../index.js";
```

Comment:
No need to handle cjs/mjs here?

Reply:
I don't think `/index` would be found as `/index.cjs` or `/index.mjs`, which makes this a non-change [except maybe in the case that index was a folder].
If we are building tests with tsc-multi and renaming (pattern we are going away from), then this would be renamed. If building only the product code with tsc-multi and renaming, then I think this was already broken.
Nothing broken on my local system when this was run for client group.